source/PaNET.owl

```diff
@@ -416,7 +364,6 @@
 
     <owl:Class rdf:about="http://purl.org/pan-science/PaNET/PaNET00401">
         <owl:equivalentClass rdf:resource="http://purl.org/pan-science/PaNET/PaNET01106"/>
-        <rdfs:subClassOf rdf:resource="http://purl.org/pan-science/PaNET/PaNET00004"/>
```

Comment:
There seems to be a number of changes that don't match the earlier change (commit 9e1ad5592d).

As an example, this change removes `defined by functional dependence` (`PaNET:00004`) as a parent of `obtain spatial map` (`PaNET:00401`), which seems like a structural change to the PaNET ontology.

Such changes seem to be inconsistent with the goal of simply removing the individuals.

This might be the result of an existing inconsistency between the current `.csv` file and the corresponding generated `.owl` file.  If so, I suggest that this inconsistency is fixed first (as a separate commit and pull-request) and then we consider this pull-request.